smithy-openapi/src/test/resources/software/amazon/smithy/openapi/fromsmithy/examples-test.openapi.json

```diff
@@ -0,0 +1,520 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Example",
+    "version": "0000-00-00"
+  },
+  "paths": {
+    "/test/get": {
+      "patch": {
+        "operationId": "Bar",
+        "requestBody": {
+          "content": {
+            "text/plain": {
+              "schema": {
+                "$ref": "#/components/schemas/BarInputPayload"
+              },
+              "examples": {
+                "Bar_PATCH_example1": {
```

Comment:
why do the keys have `Bar` and `PATCH` prepended? All the things that need to be matched are going to be inside the context on an operation (i.e., under the path and verb), so isn't strictly necessary. Though it helps if someone is say doing Ctrl-F across the file. Though I think just operationId value is enough since it is supposed to be unique and it is better for them to not have to also think about the verb for that operation. i.e., I think the key should be `Bar_example1`.

Reply:
Redundant info fixed. Now example names are formatted to only contain operation names, like `Bar_example1`.